tensorboard/components/tf_data_selector/test/storage-utils-test.ts

```diff
@@ -28,26 +28,21 @@ describe('storageUtils', () => {
       assert.deepEqual(actual, [1]);
     });
 
-    // TODO(@stephanwlee): This test fails:
-    // expected [ 1, 2, 1461559270678 ] to deeply equal [ NaN, 1, 2, NaN, Infinity ]
-    it.skip('decodes with unexpected string', () => {
+    it('filters non-number (including inf) from decoded ids', () => {
       const actual = tf_data_selector.decodeIdArray(',1, 2,!a,Infinity');
-      assert.deepEqual(actual, [NaN, 1, 2, NaN, Infinity]);
+      assert.deepEqual(actual, [1, 2]);
```

Comment:
Do we know why this test case was previously failing and why it passes
now?

Reply:
No idea how this code was committed in the first place. With the `filter` that was added 4 months ago along with this code, NaN should have never ended up here.

Reply:
Okay. I guess it’s understandable when we didn’t actually have any
system for _running_ these tests. :-)